--- make.py_bkp	2025-02-16 04:37:28.776690232 -0600
+++ make.py	2025-02-16 04:39:18.988190138 -0600
@@ -68,6 +68,16 @@
 if ("1" == config.option("update")):
   repositories = base.get_repositories()
   base.update_repositories(repositories)
+  print(repositories)
+  print("Current Directory:", os.getcwd())
+  base.update_repositories(repositories)
+  print("Current Directory:", os.getcwd())
+  server_dir = os.path.join(os.getcwd(), "..", "server")
+  os.chdir(server_dir)
+  print("Changed Directory to:", os.getcwd())
+  os.system("patch Common/sources/constants.js < /root/patches/unlimited_license.patch")
+  print("Updated LICENSE_CONNECTIONS and LICENSE_USERS to 99999 in constants.js")
+  exit(0)

 base.configure_common_apps()
