diff --git a/make.py b/make.py
index fe5b572..29d69d0 100755
--- a/make.py
+++ b/make.py
@@ -14,6 +14,8 @@ import build_server
 import deploy
 import make_common
 import develop
+import os
+import re

 # parse configuration
 config.parse()
@@ -44,7 +46,6 @@ if ("1" != base.get_env("OO_RUNNING_BRANDING")) and ("" != config.option("brandi
     base.set_env("OO_RUNNING_BRANDING", "1")
     base.set_env("OO_BRANDING", config.option("branding"))
     base.cmd_in_dir(branding_dir + "/build_tools", "python", ["make.py"])
-    exit(0)

 # correct defaults (the branding repo is already updated)
 config.parse_defaults()
@@ -54,8 +55,28 @@ base.check_build_version(base_dir)
 # update
 if ("1" == config.option("update")):
   repositories = base.get_repositories()
+  print(repositories)
+  print("Current Directory:", os.getcwd())
   base.update_repositories(repositories)
-
+  print("Patch..")
+  print("Current Directory:", os.getcwd())
+  server_dir = os.path.join(os.getcwd(), "..", "server")
+  os.chdir(server_dir)
+  print("Changed Directory to:", os.getcwd())
+  # Path to constants.js
+  constants_file = os.path.join(server_dir, "Common/sources/constants.js")
+
+  # Read and update the file
+  with open(constants_file, "r") as file:
+          content = file.read()
+  # Replace values using regex
+  content = re.sub(r"exports.LICENSE_CONNECTIONS\s*=\s*\d+;", "exports.LICENSE_CONNECTIONS = 99999;", content)
+  content = re.sub(r"exports.LICENSE_USERS\s*=\s*\d+;", "exports.LICENSE_USERS = 99999;", content)
+
+  # Write back updated content
+  with open(constants_file, "w") as file:
+          file.write(content)
+  print("Updated LICENSE_CONNECTIONS and LICENSE_USERS to 99999 in constants.js")
 base.configure_common_apps()

 # developing...